src/event_logger.cpp

```diff
@@ -0,0 +1,101 @@
+#include "event_logger.hpp"
+
+#include "logger.hpp"
+
+#include <sdbusplus/asio/property.hpp>
+#include <sdbusplus/server.hpp>
+
+namespace vpd
+{
+std::unordered_map<types::SeverityType, std::string>
```

Comment:
Mark them static to limit the scope to this file?

Reply:
m_severityMap is declared as static member.